nncf/common/graph/graph.py

```diff
@@ -608,3 +620,11 @@ def get_nx_edge(self, node_u: NNCFNode, node_v: NNCFNode):
 
     def get_nodes_count(self):
         return self._nx_graph.number_of_nodes()
+
+    def get_edge(self, from_node: NNCFNode, to_node: NNCFNode) -> NNCFGraphEdge:
```

Comment:
Could you please add a docstring for this method?

Reply:
Done